	- pass CFLAGS manually to CC calls, since using the $(M)
	  from top-level Makefile would lead to double CFLAGS passing
	  in implicit rules
--- maradns-2.0.13.orig/tcp/Makefile	2010-09-09 22:21:09.000000000 +0200
+++ maradns-2.0.13/tcp/Makefile	2015-10-22 15:46:26.000000000 +0200
@@ -17,7 +17,7 @@ OBJECTS=$(JOBJS) $(MHOBJS) $(MJOBJS) $(D
 	$(TCPOBJS)
 EXECS=getzone zoneserver fetchzone
 
-FLAGS = -DAUTHONLY
+FLAGS = -DAUTHONLY -DIPV6
 
 all: $(EXECS)
 
@@ -25,11 +25,11 @@ clean:
 	rm -f core $(EXECS) $(TCPOBJS) *.exe
 
 getzone: getzone.c $(OBJECTS) 
-	$(CC) $(FLAGS) -o getzone getzone.c $(OBJECTS)
+	$(CC) $(CFLAGS) $(FLAGS) -o getzone getzone.c $(OBJECTS)
 
 fetchzone: fetchzone.c $(OBJECTS) 
-	$(CC) $(FLAGS) -o fetchzone fetchzone.c $(OBJECTS) 
+	$(CC) $(CFLAGS) $(FLAGS) -o fetchzone fetchzone.c $(OBJECTS) 
 
 zoneserver: zoneserver.c zoneserver_en.h $(OBJECTS)
-	$(CC) $(FLAGS) -DVERSION=\"$(VERSION)\" -o zoneserver zoneserver.c $(OBJECTS) $(RNGOBJS)
+	$(CC) $(CFLAGS) $(FLAGS) -DVERSION=\"$(VERSION)\" -o zoneserver zoneserver.c $(OBJECTS) $(RNGOBJS)
 
